third_party/intel/lib/TritonGENToLLVM/TritonGENToLLVMPass.cpp

```diff
@@ -49,16 +49,39 @@ using namespace mlir::triton::gpu;
 // Helper Functions
 //===----------------------------------------------------------------------===//
 
-static intel::AttributeList
-getAttrList(const intel::AttrBuilder &funcAttrBuilder,
-            ArrayRef<NamedAttrList> paramAttrs = {}) {
+static void addToAttrList(intel::AttributeList &functionAttrs,
+                          llvm::ArrayRef<NamedAttrList> paramAttrs) {
+  functionAttrs.addParamAttributes(paramAttrs);
+}
+
+intel::AttributeList createFunctionAttributes(
+    const ArrayRef<
+        std::pair<llvm::Attribute::AttrKind, std::optional<uint64_t>>>
+        attributes,
```

Comment:
```suggestion
    ArrayRef<
        std::pair<llvm::Attribute::AttrKind, std::optional<uint64_t>>>
        attributes,
```
This is usually passed this like this

Reply:
ok... removed `const`